pkg/lightning/backend/external/onefile_writer.go

```diff
@@ -203,5 +308,21 @@ func (w *OneFileWriter) closeImpl(ctx context.Context) (err error) {
 		err = err2
```

Comment:
ditto for L300
```suggestion
        w.logger.Error("Close stat writer failed", zap.Error(err2))
		err = err2
```

Reply:
let's fix it in the master, and pick it to here